src/planner/kinodynamic/KinodynamicTimer.cpp

```diff
@@ -0,0 +1,273 @@
+#include "aikido/planner/kinodynamic/KinodynamicTimer.hpp"
+#include <dart/dart.hpp>
+#include <aikido/common/Spline.hpp>
+#include <aikido/common/StepSequence.hpp>
+
+#include "Path.h"
+#include "Trajectory.h"
+
+namespace aikido {
+namespace planner {
+namespace kinodynamic {
+
+//==============================================================================
+std::unique_ptr<aikido::trajectory::Spline>
+createSplineFromWaypointsAndConstraints(
+    const std::list<Eigen::VectorXd>& waypoints,
```

Comment:
Why a `list` over a `vector`?

Reply:
Because the external codes take in a `list` of `Eigen::VectorXd`. If we use `vector`, we will either have to do an extra conversion or hack the external code.